mne/io/edf/tests/test_edf.py

```diff
@@ -125,10 +125,14 @@ def test_edf_data():
         read_raw_edf(broken_fname, exclude=raw.ch_names[:132], preload=True)
 
 
-@pytest.mark.filterwarnings('ignore::RuntimeWarning')
 def test_duplicate_channel_labels_edf():
     """Test reading edf file with duplicate channel names."""
-    raw = read_raw_edf(duplicate_path, preload=True)
+    # reading files with dupiicate names should generate a runtime warning
```

Comment:
no need for the comment

Reply:
there's enough with the match. If you want to make it more clear, I would change the name `duplicate_path` for something that is clear with the match. 

ie:
```py
with pytest.warns(RuntimeWarning, match='Channel names are not unique'):
￼	  raw = read_raw_edf(edf_with_duplicated_channels_path, preload=True)
```